### PR TITLE
Update NPM Dependencies

### PR DESCRIPTION
**Pull request by christophertino**
This PR updates all npm packages and makes the following changes: 

+ webpack refactor (add eslint-loader)
+ replace .eslintrc.json with eslintrc.js
+ replace .babelrc with babel.config.js
+ build Cliqz content-script-bundle.js from browser-core
+ fix ESLint errors due to updated lint rules

### CODE REVIEW

File: app/content-scripts/content_script_bundle.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+/**
+ * @namespace CliqzContentScript
+ */
+
+import 'browser-core/build/core/content';
```
Comment (remusao) [inline app/content-scripts/content_script_bundle.js]: Shouldn't this be including `browser-core/build/core/content-script.bundle` instead?

File: app/panel-android/components/content/Accordion.jsx [hunk 2]
```diff
@@ -172,10 +174,18 @@ export default class Accordion extends React.Component {
 				<div className={`accordionTitle ${this.state.isActive ? 'active' : ''}`} style={titleStyle} onClick={this.toggleContent}>
 					<h2>{this.props.name}</h2>
 					<p>
-						<span className="total-trackers">{this.props.numTotal} TRACKERS</span>
-						{!!this.props.numBlocked &&
-							<span className="blocked-trackers">{this.props.numBlocked} Blocked</span>
-						}
+						<span className="total-trackers">
+							{this.props.numTotal}
+							{' '}
```
Comment (jsignanini) [inline app/panel-android/components/content/Accordion.jsx]: @christophertino probably better to replace this with `&nbsp;`?

File: app/panel-android/components/content/Accordion.jsx [hunk 3]
```diff
@@ -172,10 +174,18 @@ export default class Accordion extends React.Component {
...
+							TRACKERS
+						</span>
+						{!!this.props.numBlocked && (
+							<span className="blocked-trackers">
+								{this.props.numBlocked}
+								{' '}
```
Comment (jsignanini) [inline app/panel-android/components/content/Accordion.jsx]: @christophertino same here, replace this with &nbsp;?

File: app/panel-android/components/content/TrackersChart.jsx [hunk 4]
```diff
@@ -31,7 +31,9 @@ class TrackersChart extends React.Component {
 			<div className="trackers-chart">
 				<ChartSVG paths={this.props.paths} radius={this.state.config.radius} />
 				<p className="trackers-num">
-					<span>{this.props.num}</span> <span>Trackers found</span>
+					<span>{this.props.num}</span>
+					{' '}
```
Comment (jsignanini) [inline app/panel-android/components/content/TrackersChart.jsx]: @christophertino same here, replace this with `&nbsp;`?

File: app/panel/components/Blocking/BlockingHeader.jsx [hunk 5]
```diff
@@ -241,7 +250,9 @@ class BlockingHeader extends React.Component {
 				<div className="row align-middle">
 					<div className="columns">
 						<div className="title">
-							{ globalBlocking ? t('settings_global_blocking') : t('blocking_trackers') } <Link to="/settings/globalblocking" className="gear-icon" />
+							{ globalBlocking ? t('settings_global_blocking') : t('blocking_trackers') }
+							{' '}
```
Comment (jsignanini) [inline app/panel/components/Blocking/BlockingHeader.jsx]: @christophertino same here, replace this with `&nbsp;`?

File: app/panel/components/Blocking/Tracker.jsx [hunk 6]
```diff
@@ -248,7 +261,11 @@ class Tracker extends React.Component {
 		const cssClass = `trk-cliqz-stat trk-cliqz-stat-${type}s-count`;
 
 		return (
-			<span className={cssClass}>{count} {label}</span>
+			<span className={cssClass}>
+				{count}
+				{' '}
```
Comment (jsignanini) [inline app/panel/components/Blocking/Tracker.jsx]: @christophertino same here, replace this with `&nbsp;`?

File: app/panel/components/Settings.jsx [hunk 7]
```diff
@@ -128,12 +138,13 @@ class Settings extends React.Component {
 	/**
 	 * Hide alert in 3 sec. after it has been shown.
 	 */
-	hideToast = debounce(function () {
+	hideToast = debounce(function() { // eslint-disable-line react/sort-comp
```
Comment (jsignanini) [inline app/panel/components/Settings.jsx]: @christophertino why do we need this `eslint-disable-line react/sort-comp` here?

File: src/background.js [hunk 8]
```diff
@@ -1817,7 +1817,7 @@ function init() {
 									return account.getTheme(conf.current_theme);
 								}
 							});
-					} else if (globals.JUST_INSTALLED) {
+					} if (globals.JUST_INSTALLED) {
```
Comment (jsignanini) [inline src/background.js]: @christophertino should these `if (...) { return...; }` each be in a new line?

**Comment by jsignanini** [comment]
@christophertino also noticed node `>=8.15.0` is required by the `jsdoc@3.6.2` dependency, so we either have to update `.nvmrc` to reflect this or (preferably?) downgrade the dependency to work with `lts/carbon`.

**Comment by christophertino** [comment]
@jsignanini Changes made, please have a look

**Pull request merged**